src/Libm.jl

```diff
@@ -1,5 +1,17 @@
 module Libm
 
-# package code goes here
+include("erf.jl")
 
-end # module
+
+# Utils
+
+# Get the more significant 32 bit int from a double
+function GET_HIGH_WORD(d::Float64)
```

Comment:
In general, we should probably try to stick to the convention of constants in uppercase, types in upper camel case, and functions and variables in lowercase.